odoo/addons/base/tests/test_ir_mail_server.py

```diff
@@ -138,6 +140,23 @@ def test_mail_server_priorities(self):
         self.assertEqual(mail_server, self.server_notification, 'Should take the notification email')
         self.assertEqual(mail_from, 'notifications@test.com')
 
+        # this mail server can not be used for a specific email address and 2 domain names
+        self.server_user.from_filter = "domain1.com, specific_user@test.com, domain2.com"
+        mail_server, mail_from = self.env['ir.mail_server']._find_mail_server(email_from='"Test" <specific_user@test.com>')
+        self.assertEqual(mail_server, self.server_user, "The entire email match, the server should be used")
+        self.assertEqual(mail_from, '"Test" <specific_user@test.com>')
+        mail_server, mail_from = self.env['ir.mail_server']._find_mail_server(email_from='"Example" <test@domain2.com>')
+        self.assertEqual(mail_server, self.server_user, "The domain, the server should be used")
```

Comment:
Missing a word in the sentence? "The domain matches" ?
(Same for the other one below)

Reply:
good catch